### PR TITLE
Align Price Calculator layout page to the grid

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Align Price Calculator layout page to the 14-col grid to match Figma

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4829** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4829?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/PurchaseFormV2.css.ts [hunk 1]
```diff
@@ -6,6 +6,12 @@ const CONTENT_MAX_WIDTH = '23rem'
 export const centered = style({
   maxWidth: CONTENT_MAX_WIDTH,
   marginInline: 'auto',
+  ...responsiveStyles({
+    lg: {
+      width: `max(${CONTENT_MAX_WIDTH}, 100%)`,
+      maxWidth: 'unset',
```
Comment (gustaveen) [inline apps/store/src/features/priceCalculator/PurchaseFormV2.css.ts:12]: Make sure the form doesn't get too narrow on smaller screens

File: packages/ui/src/theme/theme.ts [hunk 2]
```diff
@@ -20,6 +20,9 @@ export const theme = {
     focus: `0 0 0 2px ${colors.textPrimary}`,
     focusAlt: `0 0 0 2px ${colors.signalBlueElement}`,
   },
+  width: {
+    layoutMax: '90rem',
```
Comment (gustaveen) [inline packages/ui/src/theme/theme.ts:24]: We have a constant for this in `GridLayout` for the max width of the site, but we might as well put it in the theme

**Review by alebedev — APPROVED** [review]

**Comment by gustaveen** [comment]
### Merge activity

* **Oct 7, 5:07 AM EDT**: @gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4829).

**Pull request merged**